data-process/docker/base/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:20.04
```

Comment:
Can we use [`python:3.10.13`](https://hub.docker.com/layers/library/python/3.10.13/images/sha256-68763c64e640977ad219a74dde8c445f6fac5d2a229c4e01058a88fcdab2249c?context=explore)  as the base image? 

Please refer to this Dockerfile
https://github.com/kubeagi/arcadia/blob/main/llms/Dockerfile.server

Reply:
fix this in the future